modules/stanford_soe_helper_collection_layouts/stanford_soe_helper_collection_layouts.info

```diff
@@ -7,22 +7,34 @@ project = stanford_soe_helper_collection_layouts
 dependencies[] = bean
 dependencies[] = context
 dependencies[] = ctools
+dependencies[] = ds
 dependencies[] = features
 dependencies[] = image
+dependencies[] = options
 dependencies[] = pathauto
+dependencies[] = stanford_magazine
+dependencies[] = stanford_news_extras
+dependencies[] = stanford_people_spotlight
 dependencies[] = stanford_soe_helper_collection
 dependencies[] = stanford_soe_helper_collection_views
 dependencies[] = stanford_soe_helper_magazine
 dependencies[] = strongarm
+dependencies[] = taxonomy
 dependencies[] = views
 dependencies[] = xmlsitemap
+features[context][] = stanford_magazine_collection_article_engineering
 features[context][] = stanford_magazine_collection_engineering
 features[context][] = stanford_magazine_collection_list_page
 features[ctools][] = context:context:3
 features[ctools][] = strongarm:strongarm:1
 features[features_api][] = api:2
 features[variable][] = pathauto_node_stanford_collection_pattern
```

Comment:
@minorwm Just checking: do you need this variable: pathauto_node_stanford_collection_pattern

Reply:
@cjwest this is needed